sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -1156,9 +1185,11 @@ private async Task RunAsync(CancellationToken cancellationToken)
                 {
                     tokenSource.Cancel();
                     await processingTask.ConfigureAwait(false);
+                    Logger.StoppedPartitionProcessing(partitionId, reason);
                 }
                 catch (Exception ex) when (ex is TaskCanceledException || ex is OperationCanceledException)
                 {
+                    Logger.StopPartitionProcessingError(partitionId, reason, ex.Message);
```

Comment:
I think we should not log an error when we get a `TaskCanceled` or `OperationCanceled` exception. These are expected and do not mean something bad has happened.